### ISSUE TITLE
Modular generate do not import source tables file

### ISSUE THREAD

**Issue by NikitaVasin** [simolus3/moor#2263]
```
//--- DataBase file ------

import 'database.drift.dart';

@DriftDatabase(
  tables: [
    Companies,
  ],
)
class Database extends $Database {
  Database(super.e);

  @override
  int get schemaVersion => 1;
}

//----- Other module file -----

@DataClassName('Company')
class Companies extends Table with BaseDriftTable {
  TextColumn get name => text().named('name')();
}

mixin BaseDriftTable on Table {
  TextColumn get id => text().clientDefault(() => const Uuid().v1())();
}

//-----Generated module file ------

// ignore_for_file: type=lint
import 'package:drift/drift.dart' as i0;
import 'package:drift_test/common/data/second_module.drift.dart' as i1;
import 'package:uuid/uuid.dart' as i2;

//! missing source file import
//! import 'package:drift_test/common/data/second_module.dart';

class $CompaniesTable extends Companies
    with i0.TableInfo<$CompaniesTable, i1.Company> { /* ... */ }

```

**Comment by NikitaVasin** [comment]
build.yaml file
```
targets:
  $default:
    builders:
      drift_dev:
        enabled: false
      drift_dev:analyzer:
        enabled: true
      drift_dev:modular:
        enabled: true
```

**Comment by NikitaVasin** [comment]
I checked out the develop branch and it didn't reproduce, but there was a problem importing the parent dataclass. Apparently, it is enough to simply add an alias to the class

```
// --- database.dart ---
import 'database.drift.dart';

@DriftDatabase(
  tables: [
    Companies,
  ],
)
class Database extends $Database {
  Database(super.e);

  @override
  int get schemaVersion => 1;
}

//--- second_module.dart ---

import 'package:drift/drift.dart';
import 'package:uuid/uuid.dart';

abstract class BaseDataClass extends DataClass {
  const BaseDataClass();
  abstract final String id;
}

@DataClassName('Company', extending: BaseDataClass)
class Companies extends Table with BaseDriftTable {
  TextColumn get name => text().named('name')();
}

mixin BaseDriftTable on Table {
  TextColumn get id => text().clientDefault(() => const Uuid().v1())();
}

// ---- second_module.drift.dart ----

class Company extends BaseDataClass implements i1.Insertable<i0.Company> {
  final String id;
  final String name;
  const Company({required this.id, required this.name});

// ...

}

```

BaseDataClass -> i2.BaseDataClass

**Comment by simolus3** [comment]
Thanks for the report! The first issue sounds like a duplicate of #2260, but the one about data classes not having the right extend was another problem. Fixed in 53c14cd1aa76306c29e5491e9b59cae2caf4d16c!
